pipeline/00-ingest.R

```diff
@@ -285,12 +287,12 @@ training_data_fil <- training_data_klg %>%
     ),
     # Kludge sale validation flags based on raw price for sales added later
```

Comment:
@wagnerlmichael You can boot this entire section (and everything else to do with `sv_added_later`). It was a kludge to solve a temporary problem.